yt_dlp/extractor/redgifs.py

```diff
@@ -102,6 +129,8 @@ class RedGifsIE(RedGifsBaseInfoExtractor):
             'like_count': int,
             'categories': list,
             'age_limit': 18,
+            'ie_key': 'RedGifs',
```

Comment:
```suggestion
```

Reply:
This is in here because the test fails if I take it out:
```======================================================================
FAIL: test_RedGifs (__main__.TestDownload):
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test/test_download.py", line 214, in test_template
    expect_info_dict(self, tc_res_dict, tc.get('info_dict', {}))
  File "/tmp/yt-dlp/test/helper.py", line 265, in expect_info_dict
    self.assertFalse(
AssertionError: {'ie_key'} is not false : Missing keys in test definition: ie_key
```
It appears to be explicitly put in earlier:
```python
        return {
            'id': video_id,
            'webpage_url': f'https://redgifs.com/watch/{video_id}',
            'ie_key': RedGifsIE.ie_key(),
```

There are a bunch of other extractors that also set `ie_key` as well.

Reply:
It should be `extractor_key`